Core/clim-basic/graphics.lisp

```diff
@@ -146,8 +146,9 @@
 			     (line-style-unit line-style))
 		   :thickness (or line-thickness
 				  (line-style-thickness line-style))
-		   :dashes (or line-dashes
-			       (line-style-dashes line-style))
+		   :dashes (if dashes-p
+                               line-dashes
+                               (line-style-dashes line-style))
```

Comment:
Indentation looks wonky. Maybe untabify the method?

Reply:
willdo

Reply:
I've detabified the file in a separate (cosmetic) commit.